packages/jumpstarter-driver-flashers/oci_bundles/build_bundle.sh

```
@@ -12,12 +12,16 @@ echo "Building and pushing ${FLASHER_OCI_CONTAINER}"
 set -x
 
 cd "${BUNDLE_FILES}"
+MANIFESTS=
+for file in $(ls -1 *.yaml); do
+	MANIFESTS="${MANIFESTS} ${file}:application/yaml "
+done
 DATA_FILES=
-for file in $(find ./data -type f); do
+for file in $(find ./data -type f -prune -a -not -name .gitkeep); do
```

Shellcheck (0.10.0)

[warning] For loops over find output are fragile. Use find -exec or a while read loop.

(SC2044)